lang/cs/Org.Apache.REEF.Network/Naming/NameCache.cs

```diff
@@ -0,0 +1,76 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.Collections.Specialized;
+using System.Net;
+using System.Runtime.Caching;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Network.Naming
+{
+    internal class NameCache
+    {
+        private readonly MemoryCache _cache;
+        private readonly double _expirationDuration;
```

Comment:
done..usage in cache requires it to be double. I have added comment
regarding this.

On Mon, Jul 27, 2015 at 7:28 AM, Markus Weimer <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.Network/Naming/NameCache.cs
> <https://github.com/apache/incubator-reef/pull/318#discussion_r35541145>:
>
> > + * specific language governing permissions and limitations
> > + * under the License.
> > + */
> > +
> > +using System;
> > +using System.Collections.Specialized;
> > +using System.Net;
> > +using System.Runtime.Caching;
> > +using Org.Apache.REEF.Tang.Annotations;
> > +
> > +namespace Org.Apache.REEF.Network.Naming
> > +{
> > +    internal class NameCache
> > +    {
> > +        private readonly MemoryCache _cache;
> > +        private readonly double _expirationDuration;
>
> Why a double? Also, please document the units used. Is it seconds?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/318/files#r35541145>.
>